cime/scripts/lib/CIME/SystemTests/pgn.py

```diff
@@ -570,53 +570,57 @@ def run_phase(self):
                 
                 iinst += 1
 
+        #cloud generation
 
-        if(self._case.get_value("GENERATE_BASELINE")):
+        if(debug):
+            print("PGN_INFO: cloud generation-gen base")
```

Comment:
logger.debug(...) is what you want here

Reply:
CIME already supports --debug flag for all tools.

Reply:
Thanks! I tried `logger.debug` in the past and it didn't work at that time. perhaps I wasn't using it correctly. Would you please point me some place where it is used in the scripts or tests? Is `--debug`  flag a `create_test` scripts's argument?

Reply:
@singhbalwinder , all CIME tools should support --debug. You can grep for "logger.debug".